tools/gen-intrinsics-specs.py

```diff
@@ -65,6 +67,15 @@ def rst_literal_quote(mapping, source_syntax):
 
     >>> rst_literal_quote('', 'markdown')
     ''
+
+    >>> rst_literal_quote('a; b', 'pdf')
+    '`a`\n\n`b`'
```

Comment:
This should be similar to the rst output. the `;` separator would make the final output look like the following:

```
<three times `>\na\nb\n<three times `>
```

Basically we need to use the ``` to wrap the sub string a and b, each with a new line, so that in the final rendering they will look as follow:

```
a
b
```

Reply:
Is the output for "a b" meant to be "<three times `>a b<three times `>" then?

Reply:
yep, it could be that, with newlines to separate the `<three times that simbol>` from `a b`:

```
<three times `>\na b\n<three times `>
```